src/main/java/uk/gov/hmcts/cmc/claimstore/repositories/ClaimRepository.java

```diff
@@ -18,10 +18,10 @@
 public interface ClaimRepository {
 
     @SuppressWarnings("squid:S1214") // Pointless to create class for this
-    String SELECT_FROM_STATEMENT = "SELECT * FROM claim";
+        String SELECT_FROM_STATEMENT = "SELECT * FROM claim";
```

Comment:
Autoformatting will align these correctly if you move the {{// pointless}} comments to their own lines. Don't bother if it's the only change

Reply:
Good to know. I will update it :)

Reply:
done